website/docs/guides/core-to-cloud-3.md

```diff
@@ -0,0 +1,188 @@
+- [Development tools](https://docs.getdbt.com)/guides/core-to-cloud-3?step=4) 
+- [Orchestration](https://docs.getdbt.com/guides/core-to-cloud-3?step=5)
+- [dbt Mesh](https://docs.getdbt.com/guides/core-to-cloud-3?step=6)
+- [dbt Semantic Layer](https://docs.getdbt.com/guides/core-to-cloud-3?step=7)
+- [dbt Explorer](https://docs.getdbt.com/guides/core-to-cloud-3?step=8)
+- [What's next?](https://docs.getdbt.com/guides/core-to-cloud-3?step=9)
+
+## Adapters and connections
+
+In dbt Cloud, you can natively connect to your data platform and test its [connection](/docs/connect-adapters) with a click of a button. This is especially useful for users who are new to dbt Cloud or are looking to streamline their connection setup. Here are some tips and caveats to consider:
+
+### Tips
+- Manage [dbt versions](/docs/dbt-versions/upgrade-dbt-version-in-cloud) and ensure team synchronization with dbt Cloud's one-click feature, eliminating the hassle of manual updates and version discrepancies.
+- dbt Cloud supports a whole host of [cloud providers](/docs/cloud/connect-data-platform/about-connections), including Snowflake, Databricks, BigQuery, Fabric, and Redshift (to name a few).
+- Use [Extended Attributes](/docs/deploy/deploy-environments#extended-attributes) to set a flexible [profiles.yml](/docs/core/connect-data-platform/profiles.yml) snippet in your dbt Cloud Environment settings. It gives you more control over environments (both deployment and development) and extends how dbt Cloud connects to the data platform within a given environment.
+  - For example, if you have a field in your `profiles.yml` that you’d like to add to the dbt Cloud adapter user interface, you can use Extended Attributes to set it.
+
+### Caveats
+- Not all parameters are available for adapters.
+- One project can only use one warehouse type.
+
+## Development tools
+
+dbt Cloud empowers data practitioners to develop in the tool of their choice. It ships with a [dbt Cloud CLI](/docs/cloud/cloud-cli-installation) (local) or [dbt Cloud IDE](/docs/cloud/dbt-cloud-ide/develop-in-the-cloud) (browser-based) to build, test, run, and version control your dbt projects.
+
+Both development tools are tailored to suit different roles and preferences within your team. To streamline your team’s workflow, it's important to know who is likely to prefer the dbt Cloud IDE and who might lean towards the dbt Cloud CLI. This section is a little different to the others and aims to clarify these preferences.
+
+### dbt Cloud IDE
+A single web-based interface for building, testing, running, and version-controlling dbt projects. It compiles dbt code into SQL and executes it directly on your database.
+
+**Who might prefer the dbt Cloud IDE?**
+
+- New dbt users or those transitioning from other tools who appreciate a more guided experience through a browser-based interface.
+- Team members focused on development speed and convenience, including analysts and less technical stakeholders.
```

Comment:
A lot of the language here makes the IDE seem like the ideal tool for the new user or junior roles, but some advanced users are going to enjoy using it as well and we should be mindful of that. We might want to call them out in a positive way so they don't feel like using the IDE is somehow making them less than a senior analyst.

"Makes it fast and easy for the most advanced analytics engineers to build and test changes" or something like that.